### PR TITLE
support flux 0.9 and update docs

### PR DESCRIPTION
**Pull request by saada**
- flux 0.9.0 and 0.9.1
- new [description](https://github.com/weaveworks/flux2-openshift/blob/flux-0.9/templates/clusterserviceversion.yaml#L252-L254)
- add [prerequisites](https://github.com/weaveworks/flux2-openshift/blob/flux-0.9/templates/clusterserviceversion.yaml#L329-L366) section to guide folks on SCC requirements that Flux added recently
Screen Shot 2021-03-12 at 8 34 14 AM
Screen Shot 2021-03-12 at 8 34 43 AM

### CODE REVIEW

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @saada

File: templates/clusterserviceversion.yaml [hunk 1]
```diff
@@ -326,11 +327,43 @@ spec:
         version: v1beta1
         description: Receiver
   description: |-
-    Open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
+    The GitOps family of projects.
+    [Flux](https://fluxcd.io/) is a set of continuous and progressive delivery solutions for Kubernetes, and they are open and extensible.
+    A collection of tools for keeping Kubernetes clusters in sync with sources of configuration (like Git repositories), and automating updates to configuration when there is new code to deploy.
```
Comment (stefanprodan) [inline templates/clusterserviceversion.yaml:332]:
```suggestion
[Flux](https://fluxcd.io/) is Continuous Delivery solution for Kubernetes.
Flux is constructed with the [GitOps Toolkit](https://toolkit.fluxcd.io/components/), a set of composable APIs and specialized tools for keeping Kubernetes clusters in sync with sources of configuration (like Git & Helm repositories), and automating updates to configuration when there is new code to deploy. 
```

File: templates/clusterserviceversion.yaml [hunk 2]
```diff
@@ -326,11 +327,42 @@ spec:
...
   description: |-
-    Open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
+    [Flux](https://fluxcd.io/) is a Continuous Delivery solution for Kubernetes.
+    Flux is constructed with the [GitOps Toolkit](https://toolkit.fluxcd.io/components/), a set of composable APIs and specialized tools for keeping Kubernetes clusters in sync with sources of configuration (like Git & Helm repositories), and automating updates to configuration when there is new code to deploy.
 
-    [Learn more](https://fluxcd.io/)
+    Flux is a CNCF incubation project.
 
-    NOTE: If you're using a CNI that requires network policies, you may need to [add Flux's policies](https://github.com/fluxcd/flux2/tree/main/manifests/policies).
+    ### Pre-requisites
```
Comment (stefanprodan) [inline templates/clusterserviceversion.yaml:335]:
```suggestion
    ### Prerequisites
```

File: templates/clusterserviceversion.yaml [hunk 3]
```diff
@@ -326,11 +327,42 @@ spec:
         version: v1beta1
         description: Receiver
   description: |-
-    Open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
+    [Flux](https://fluxcd.io/) is a Continuous Delivery solution for Kubernetes.
+    Flux is constructed with the [GitOps Toolkit](https://toolkit.fluxcd.io/components/), a set of composable APIs and specialized tools for keeping Kubernetes clusters in sync with sources of configuration (like Git & Helm repositories), and automating updates to configuration when there is new code to deploy.
 
-    [Learn more](https://fluxcd.io/)
+    Flux is a CNCF incubation project.
```
Comment (stefanprodan) [inline templates/clusterserviceversion.yaml:333]:
```suggestion
    Flux is a CNCF incubation project made for:

    - **cluster operators** who automate provision and configuration of clusters
    - **platform engineers** who build continuous delivery for developer teams
    - **app developers** who rely on continuous delivery to get their code live
    
    Features:

    - Source configuration from Git and Helm repositories, and S3-compatible buckets (e.g., Minio)
    - Kustomize and Helm support
    - Event-triggered and periodic reconciliation
    - Integration with Kubernetes RBAC
    - Health assessment (clusters and workloads)
    - Dependency management (infrastructure and workloads)
    - Secrets management (builtin support for Mozilla SOPS and Cloud KMS)
    - Alerting to external systems (webhook senders)
    - External events handling (webhook receivers)
    - Automated container image updates to Git (image scanning and patching)
    - Policy-driven validation (OPA, Kyverno, admission controllers)
    - Seamless integration with Git providers (GitHub, GitLab, Bitbucket)
    - Interoperability with Cluster API (CAPI) providers
```

**Review by stefanprodan — APPROVED** [review]

**Review by scottrigby — COMMENTED** [review]
This looks great 👏 

I would love to update to this new Flux project description https://github.com/fluxcd/flux2/discussions/620 but that is not approved yet, so this is perf 👌

**Pull request merged**